DOCS.md

```diff
@@ -106,58 +110,108 @@ steps:
+| `context`       | path to context for building the image                             | `true`   | `.`               | `PARAMETER_CONTEXT`<br>`KANIKO_CONTEXT`                        |
+| `dockerfile`    | path to the file for building the image                            | `true`   | `Dockerfile`      | `PARAMETER_DOCKERFILE`<br>`KANIKO_DOCKERFILE`                  |
+| `dry_run`       | enable building the image without publishing                       | `false`  | `false`           | `PARAMETER_DRY_RUN`<br>`KANIKO_DRY_RUN`                        |
+| `event`         | event generated for build                                          | `true`   | **set by Vela**   | `PARAMETER_EVENT`<br>`KANIKO_EVENT`<br>`VELA_BUILD_EVENT`      |
+| `labels`        | unique labels to add to the image                                  | `false`  | `N/A`             | `PARAMETER_LABELS`<br>`KANIKO_LABELS`                          |
+| `log_level`     | set the log level for the plugin                                   | `true`   | `info`            | `PARAMETER_LOG_LEVEL`<br>`KANIKO_LOG_LEVEL`                    |
+| `mirror`        | name of the mirror registry to use                                 | `false`  | `N/A`             | `PARAMETER_MIRROR`<br>`KANIKO_MIRROR`                          |
+| `password`      | password for communication with the registry                       | `true`   | `N/A`             | `PARAMETER_PASSWORD`<br>`KANIKO_PASSWORD`<br>`DOCKER_PASSWORD` |
+| `registry`      | name of the registry for the repository                            | `true`   | `index.docker.io` | `PARAMETER_REGISTRY`<br>`KANIKO_REGISTRY`                      |
+| `repo`          | name of the repository for the image                               | `true`   | `N/A`             | `PARAMETER_REPO`<br>`KANIKO_REPO`                              |
+| `sha`           | SHA-1 hash generated for commit                                    | `true`   | **set by Vela**   | `PARAMETER_SHA`<br>`KANIKO_SHA`<br>`VELA_BUILD_COMMIT`         |
+| `snapshot_mode` | control how to snapshot the filesystem. - options (full|redo|time) | `false`  | `N/A`             | `PARAMETER_SNAPSHOT_MODE`<br>`KANIKO_SNAPSHOT_MODE`            |
+| `tag`           | tag generated for build                                            | `false`  | **set by Vela**   | `PARAMETER_TAG`<br>`KANIKO_TAG`<br>`VELA_BUILD_TAG`            |
+| `tags`          | unique tags of the image                                           | `true`   | `latest`          | `PARAMETER_TAGS`<br>`KANIKO_TAGS`                              |
+| `target`        | set the target build stage for the image                           | `false`  | `N/A`             | `PARAMETER_TARGET`<br>`KANIKO_TARGET`                          |
+| `username`      | user name for communication with the registry                      | `true`   | `N/A`             | `PARAMETER_USERNAME`<br>`KANIKO_USERNAME`<br>`DOCKER_USERNAME` |
```

Comment:
I assume 
```
`PARAMETER_USERNAME`<br>`KANIKO_USERNAME`<br>`DOCKER_USERNAME` 
```
breaks pattern for backwards compatability for `DOCKER_USERNAME` ?

Reply:
Yes that's correct 🙂